clang/lib/APINotes/APINotesWriter.cpp

```diff
@@ -1096,6 +1163,27 @@ namespace {
 
       writer.write<uint8_t>(payload);
 
+      if (auto importAs = info.getImportAs()) {
```

Comment:
Another opportunity to refactor to extract the logic to write out the string to simplify the writing code.